File: QualityTest/src/main/java/Model/MetricsRequest.java
package Model;

public class MetricsRequest {

  public static final String TYPE = "METRICS";

  private String ProjectPath;

  private String HistoryPath;

  private String ProjectName;

  private String RepositoryOrigin;

  private String OrderedReleasePath;

  private String ReleasesPath;

  public MetricsRequest(String projectPath, String historyPath, String projectName,
                        String repositoryOrigin, String orderedReleasePath,
                        String releasesPath) {
    ProjectPath = projectPath;
    HistoryPath = historyPath;
    ProjectName = projectName;
    RepositoryOrigin = repositoryOrigin;
    OrderedReleasePath = orderedReleasePath;
    ReleasesPath = releasesPath;
  }

  public String getProjectPath() {
    return ProjectPath;
  }

  public void setProjectPath(String projectPath) {
    ProjectPath = projectPath;
  }

  public String getHistoryPath() {
    return HistoryPath;
  }

  public void setHistoryPath(String historyPath) {
    HistoryPath = historyPath;
  }

  public String getProjectName() {
    return ProjectName;
  }

  public void setProjectName(String projectName) {
    ProjectName = projectName;
  }

  public String getRepositoryOrigin() {
    return RepositoryOrigin;
  }

  public void setRepositoryOrigin(String repositoryOrigin) {
    RepositoryOrigin = repositoryOrigin;
  }

  public String getOrderedReleasePath() {
    return OrderedReleasePath;
  }

  public void setOrderedReleasePath(String orderedReleasePath) {
    OrderedReleasePath = orderedReleasePath;
  }

  public String getReleasesPath() {
    return ReleasesPath;
  }

  public void setReleasesPath(String releasesPath) {
    ReleasesPath = releasesPath;
  }
}

File: ServerFrontEnd/ServerFrontEnd/Operations/Metrics/MetricsRequest.cs
namespace ServerFrontEnd.Operations.Metrics;

public class MetricsRequest
{
    public String ProjectPath { get; set; }

    public String HistoryPath { get; set; }

    public String ProjectName { get; set; }

    public String RepositoryOrigin { get; set; }

    public String OrderedReleasePath { get; set; }
    
    public String ReleasesPath { get; set; }

    public MetricsRequest(string projectPath, string historyPath, string projectName, string repositoryOrigin, string orderedReleasePath, string releasesPath)
    {
        ProjectPath = projectPath;
        HistoryPath = historyPath;
        ProjectName = projectName;
        RepositoryOrigin = repositoryOrigin;
        OrderedReleasePath = orderedReleasePath;
        ReleasesPath = releasesPath;
    }
}